alert-policies/dbt-cloud/dbt-resource-failed.yml

```diff
@@ -0,0 +1,75 @@
+# Name of the alert
+name: Dbt Resource Failed 
+
+# Description and details
+description: |+
+  Generates critical alerts for dbt resources fail with an error. Dbt resources include models, snapshots, seeds and tests. 
```

Comment:
This description should be less than 240 characters

Reply:
updated